src/custom/state/price/reducer.ts

```diff
@@ -134,9 +140,16 @@ export default createReducer(initialState, (builder) =>
         quotes[chainId][sellToken] = { ...quoteInformation, ...payload }
       }
 
+      if (!state.initialQuoteLoaded) {
+        initialPriceLoadAnalytics()
+      }
```

Comment:
I think this part shouldn't be part of a reducer. You can make it part of a middleware as a side effect of getting the first price

Reply:
Done, moved to price middleware :+1: